libs/ledger-live-common/src/apps/mock.ts

```diff
@@ -99,7 +99,11 @@ export function mockListAppsResult(
       const dependencies = whitelistDependencies.includes(name)
         ? []
         : getDependencies(name);
-      const currency = findCryptoCurrency((c) => c.managerAppName === name);
+      const currency =
+        // try to find the "official" currency when possible (2 currencies can have the same manager app and ticker)
+        findCryptoCurrency((c) => c.name === name) ||
+        // Else take the first one with that manager app
+        findCryptoCurrency((c) => c.managerAppName === name);
```

Comment:
i have feeling we need to search for "managerAppName" first, then fallback on .name

Reply:
Well if we do that, for example:
- I'm looking for `name = Ethereum`.
- Looking through the list `Arbitrum = { managerAppName = "Ethereum" }` 
- `Currency = Arbitrum`
When in fact we wanted to get "official" Ethereum currency.